documentation/analytics/recipes/daily-active-users.md

```diff
@@ -1,2 +1,162 @@
 # Daily Active Users, DAU
 
+__Daily Active Users (DAUs)__ is the number of unique users that start at least one session in your app on any given day. DAU is a high level metric about the usage of your game over time. 
+
+## End Result
+
+ A DAU count is a snapshot of time, for example on 17/05/2017 a game has 10,000 DAU. [date, count]
```

Comment:
Awesome, could you please change the time format to the format we use everywhere else. YYYY-MM-DD